bitwarden_license/test/Scim.IntegrationTest/Controllers/v2/GroupsControllerTests.cs

```diff
@@ -0,0 +1,464 @@
+                    }
+                },
+                Schemas = new List<string>()
+            };
+
+            var context = await _factory.GroupsPatchAsync(organizationId, id, model);
+
+            Assert.Equal(StatusCodes.Status204NoContent, context.Response.StatusCode);
+
+            var databaseContext = _factory.GetDatabaseContext();
+            Assert.Empty(databaseContext.GroupUsers);
+            Assert.Equal(3, databaseContext.Groups.Count());
+
+            var group = databaseContext.Groups.FirstOrDefault(g => g.Id.ToString() == id);
+            Assert.Equal("Patch Display Name", group.Name);
+        }
+
+        [Fact]
+        public async Task Patch_RemoveListMembers_Success()
+        {
+            var organizationId = ScimApplicationFactory.TestOrganizationId1;
+            var id = ScimApplicationFactory.TestGroupId1;
+            var model = new ScimPatchModel
+            {
+                Operations = new List<ScimPatchModel.OperationModel>()
+                {
+                    new ScimPatchModel.OperationModel
+                    {
+                        Op = "remove",
+                        Path = "members",
+                        Value = JsonDocument.Parse($"[{{\"value\":\"{ScimApplicationFactory.TestOrganizationUserId1}\"}}]").RootElement
+                    }
+                },
+                Schemas = new List<string>()
+            };
+
+            var context = await _factory.GroupsPatchAsync(organizationId, id, model);
+
+            Assert.Equal(StatusCodes.Status204NoContent, context.Response.StatusCode);
+
+            var databaseContext = _factory.GetDatabaseContext();
+            Assert.Empty(databaseContext.GroupUsers);
+        }
+
+        [Fact]
+        public async Task Patch_NotFound()
+        {
+            var organizationId = ScimApplicationFactory.TestOrganizationId1;
+            var id = Guid.NewGuid();
+            var model = new Models.ScimPatchModel
+            {
+                Operations = new List<ScimPatchModel.OperationModel>(),
+                Schemas = new List<string>()
+            };
+
+            var context = await _factory.GroupsPatchAsync(organizationId, id.ToString(), model);
+
+            Assert.Equal(StatusCodes.Status404NotFound, context.Response.StatusCode);
+
+            var responseModel = JsonSerializer.Deserialize<ScimErrorResponseModel>(context.Response.Body, new JsonSerializerOptions { PropertyNamingPolicy = JsonNamingPolicy.CamelCase });
+            Assert.Equal(404, responseModel.Status);
+            Assert.Equal("Group not found.", responseModel.Detail);
+            Assert.Equal(new List<string> { ScimConstants.Scim2SchemaError }, responseModel.Schemas);
+
+            var databaseContext = _factory.GetDatabaseContext();
+            Assert.Equal(3, databaseContext.Groups.Count());
+            Assert.True(databaseContext.Groups.FirstOrDefault(g => g.Id == id) == null);
```

Comment:
You're not including any patch models in your request, I'm not sure you need to assert db state here.